flash_patcher/antlr/Stagefile.g4

```diff
@@ -3,11 +3,9 @@ grammar Stagefile;
 // PARSER RULES ----
 patchFile       : PATCH_FILE;
 pythonFile      : PYTHON_FILE;
-assetPackFile   : ASSET_PACK_FILE;
-root            : (patchFile | pythonFile | assetPackFile)*;
+root            : (patchFile | pythonFile)*;
 
 
 // LEXER RULES ----
-PATCH_FILE          : FILE_NAME_CHARACTER+ '.patch';
+PATCH_FILE          : FILE_NAME_CHARACTER+ ('.patch' | '.assets');
```

Comment:
Honestly this probably isn't necessary, since everything will be updated in a timely fashion, but no harm in leaving it here on the off chance somebody wants to build an old version of a mod via the commit history.

That said, I should probably update the SM63 integration tests. They're outdated to the point of not really being an accurate representation anymore.